alerts/machine-config-operator/MachineConfigDaemonPivotError.md

```diff
@@ -0,0 +1,151 @@
+# MCDPivotError
+
+## Meaning
+
+Alerts the User when the Machine Config Daemon
+is either unable to or detecs an error when
+trying to pivot OS image to a new version.
+
+>Alerts the user when an error is detected upon pivot.
+This triggers if the pivot errors are above zero for 2 minutes.
+
+This alert occurs when a node experiences
+OS update or kernel change failure(s)
+lasting over 2 minutes during a cluster upgrade.
+
+## Impact
+
+If the MCD is unable to update the OS and
+finish pivoting then this can prevent an
+OpenShift upgrade from completing.
+This can leave the cluster an a
+unstable state and further affect the operation of the cluster.
+
+## Diagnosis
+
+When a node pivots to update the OS image.
+This is done via the `rpm-ostree` service
+that logs any actions taken to the `machine-config-daemon-*` pod
+
+If the Machine Config Daemon is failing to
+update or pivot the nodes OS or Kernel
+then the first logs that should
+be checked are the `machine-config-daemon-*`
+pod logs for the cluster
+
+For the following command, replace the $DAEMONPOD variable
+with the name of your own machine-config-daemon-* pod name.
+To the erroring node.
```

Comment:
> To the erroring node.

Not sure what this means.
Also I'm not sure that _erroring_ is a word. Instead of _erroring node_, maybe _node that has an error_? _node experiencing an error_

Reply:
It is actually a word!

https://en.wiktionary.org/wiki/erroring

But I do agree it probably could be worded better